data/kestrel.txt

```diff
@@ -14,7 +14,7 @@ mission "Kestrel Testing"
 	source "Wayfarer"
 	waypoint "Umbral"
 	to offer
-		"combat rating" > 8000
+		"combat rating" > 2980
```

Comment:
To be clear, combat rating 9 is actually 8103, not 8000, so evidently Michael was looking for a round number here.

Reply:
Having it linked to a number displayed in-game makes more sense as then we can point to that number. While the 8000 was likely because it is a round number that is not an important factor in choosing the number.

Reply:
I mean, sure, but then it just might feel a bit too convenient or unnatural (in-game, as opposed to someone just looking right at the code, because 8000 maps to 8.97 rather than 9). I agree with your statement, though, it might be better overall to change the scheme here.